.gitignore

```diff
@@ -1,5 +1,6 @@
 /build
 /vendor
+.coveralls.yml
```

Comment:
@szepeviktor How can we fix these? https://github.com/predis/predis/actions/runs/6140711631/job/16660032679?pr=1397

Reply:
```diff
- .coveralls.yml,LICENSE,README.md,autoload.php,composer.json
+ LICENSE,README.md,autoload.php,composer.json
```

`.coveralls.yml` is added to the repo. It should not be ignored.
Stop exporting it in `.gitattributes`
```
.coveralls.yml export-ignore
```